solr/webapp/web/error404.html

```diff
@@ -0,0 +1,5 @@
+<p>
+  Solr is here to help.</br>
+  You must type the right path though.</br>
+  Solr will respond.
```

Comment:
If this is meant to be a haiku, which I hope it is and appreciate greatly, you have Solr in the first line as a single syllable but in the last line as two syllables. Maybe

```
Searching for Solr?
You must type the correct path.
Solr will respond.
```